packages/textmate-grammar/src/squiggle.tmLanguage.yaml

```diff
@@ -30,9 +30,11 @@ repository:
       "4":
         name: variable.other.squiggle
   let:
-    match: ^\s*(\w+)\s*=
+    match: ^\s*(?:(export)\s+)?(\w+)\s*=
```

Comment:
Fix this for dashes, too, I presume.

Reply:
Not here, this is for Squiggle variable names.